src/Adaptor/SentryAdaptor.php

```diff
@@ -104,7 +104,7 @@ public function setData(string $field, $data) : void
                 break;
             case 'level':
                 Hub::getCurrent()->configureScope(function (Scope $scope) use($data) : void {
-                    $scope->setLevel($data);
```

Comment:
Suggest we add  default level of "WARNING" in the module's config. Without doing anything other than checking out and running with this branch, there are errors emitted from sentry/Severity.php about a missing log level:

> Fatal error: Uncaught InvalidArgumentException: The "" is not a valid enum value. in /var/www/html/walletmatrix.app/vendor/sentry/sentry/src/Severity.php:76

The following config in the module's "config.yml" will sort that:

```
PhpTek\Sentry\Log\SentryLogger:
  # One of the permitted severities: DEBUG|INFO|WARNING|ERROR|FATAL
  log_level: WARNING
  dependencies:
    adaptor: %$PhpTek\Sentry\Adaptor\SentryAdaptor
```

Reply:
Completely agree that `WARNING` should be the default.